Android export broken on Windows with 3.2.4 beta4

**Godot version:**
3.2.4 beta4

**OS/device including version:**
Windows 10

**Issue description:**
When exporting to Android (APK) with custom templates i get this error.
This only happens when i export directly into the root of a drive `(eg. D:\)`. When i export to `D:\foo\` it works. 

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':copyAndRenameDebugApk'.
> java.io.IOException: The filename, directory name, or volume label syntax is incorrect

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

**Steps to reproduce:**
Create a new project, install export templates and export.

--- a/core/string/ustring.cpp
+++ b/core/string/ustring.cpp
@@ -4285,7 +4285,10 @@ bool String::is_rel_path() const {
 }
 
 String String::get_base_dir() const {
-	int basepos = find("://");
+	int basepos = find(":/");
+	if (basepos == -1) {
+		basepos = find(":\\");
+	}
 	String rs;
 	String base;
 	if (basepos != -1) {

